.github/workflows/php.yml

```diff
@@ -0,0 +1,73 @@
+name: CI FOSHttpCacheBundle
+
+env:
+  SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - php-version: '7.2'
+            symfony-version: ''
+            stability: ''
+          - php-version: '7.3'
+            symfony-version: ''
+            stability: ''
+          - php-version: '7.2'
+            symfony-version: ''
+            stability: 'dev'
+          - php-version: '7.3'
+            symfony-version: ''
+            stability: 'dev'
+          - php-version: '7.2'
+            symfony-version: '^4'
+            stability: ''
+          - php-version: '7.3'
+            symfony-version: '^4'
+            stability: ''
+          - php-version: '7.4'
+            symfony-version: '^4'
+            stability: ''
+          - php-version: '8.0'
+            symfony-version: '^4'
+            stability: ''
+          - php-version: '7.3'
+            symfony-version: '^5'
+            stability: ''
+          - php-version: '7.4'
+            symfony-version: '^5'
+            stability: ''
+          - php-version: '8.0'
+            symfony-version: '^5'
+            stability: ''
+    name: PHP ${{ matrix.php-version }} Test on Symfony ${{ matrix.symfony-version }}
+    steps:
+      - name: Pull the code
+        uses: actions/checkout@v2
+      - name: Install PHP and Composer
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          tools: composer:v2
+      - name: Check PHP Version
+        run: php -v
+      - name: Stability
+        run: composer config minimum-stability ${{ matrix.stability }}
+        if: ${{ matrix.stability }}
+      - name: Symfony version
+        run: composer require --no-update symfony/flex && composer config extra.symfony.require ${{ matrix.symfony-version}}
+        if: ${{ matrix.symfony-version }}
+      - name: Composer update
+        run: composer update --prefer-stable --prefer-lowest --prefer-dist --no-interaction
+      - name: Composer validate
+        run: composer validate --strict --no-check-lock
+      - name: Install PHPUnit
+        run: php vendor/bin/simple-phpunit install
```

Comment:
If I’m not mistaken, the PHPUnit bridge should itself install dependencies when running the tests for the first time.

Reply:
It's true :)